src/lib.rs

```diff
@@ -1,3 +1,147 @@
+//! A wrapper over [`gstd`](https://docs.gear.rs/gstd/) that provides a fluent interface to interact
+//! with the Gear Protocol.
+//!
+//! This library can be used instead of the standard library when writing Gear programs. Compared to
+//! the [`gstd`](https://docs.gear.rs/gstd/) crate, this library provides higher-level functions
+//! that allow you to write Gear programs code in a more convenient way.
+//!
+//! ### Example
+//!
+//! Consider an example where the program responds 30 blocks after sending the message.
+//!
+//! ```rust
+//! # const _: &'static str = stringify! {
+//! #![no_std]
+//! # };
+//!
+//! use gstd_fluent::{
+//!     self as builder,
+//!     gstd::{exec, msg, prelude::*},
+//! };
+//!
+//! #[no_mangle]
+//! extern "C" fn handle() {
+//!     let block_height = exec::block_height();
+//!     let payload = format!("You sent me a message in block {block_height}");
+//!
+//!     // same as `msg::send_delayed(msg::source(), payload, 30, 0)`
+//!     builder::send(msg::source(), payload.clone())
+//!         .with_delay(30)
+//!         // ^ you can customize the message here
+//!         //   e.g. specify delay, gas limit, reserved gas
+//!         .execute()
+//!         // ^ when you call `execute()` it calls the desired gstd function
+//!         .expect("Unable to reply");
+//!
+//!     // for example, you want to specify an explicit gas limit and value:
+//!     builder::send(msg::source(), payload.clone())
+//!         .with_delay(30)
+//!         // add two new lines:
+//!         .with_gas_limit(1_000_000)
+//!         .with_value(1_000) //value=0 by default
```

Comment:
Whitespace after `//` as in other comments.
```suggestion
//!         .with_value(1_000) // value=0 by default
```